resources/devices/tablet/motorola.json

```diff
@@ -76,5 +102,18 @@
             "isMobileDevice": "true",
             "isTablet": "true"
         }
+    },
+    "Motorola Xoom 2 ME": {
```

Comment:
In some cases there is just "Xoom 2 ME" in the UA, so this is a general purpose device where we can't decipher if it's the Wifi or 3G version.